src/api/status/src/server.js

```diff
@@ -54,5 +69,39 @@ service.router.get(`${process.env.PATH_PREFIX || ''}/status`, (req, res) => {
     .catch((err) => res.status(500).json({ error: err.message }));
 });
 
+// Get Home page
+service.router.get(process.env.PATH_PREFIX || '/', async (req, res) => {
+  const [telescope, satellite, totalPost, totalFeeds] = await Promise.all([
+    getGitHubData('Seneca-CDOT', 'telescope'),
+    getGitHubData('Seneca-CDOT', 'satellite'),
+    getPostsCount(),
+    getFeedCount(),
+  ]);
+  let environment = {};
+  if (
+    process.env.API_URL === 'http://localhost:3000' ||
+    process.env.API_URL === 'https://api.telescope.cdot.systems'
+  ) {
+    environment = { name: 'staging', staging: true, production: false };
```

Comment:
Just use `staging: true` in this case, and `production: true` in the other, don't bother including both, since they are the inverse of each other.

Reply:
Changed

Reply:
Changed